app/javascript/src/components/editor/Modals/TranslationKeysModal.svelte

```diff
@@ -60,27 +46,11 @@
             </div>
           {/each}
         </div>
+      {:else}
+        <p class="m-0">No keys</p>
```

Comment:
In Svelte you can also add an `else` after the `each`, which will be true if the array has no length

(Github is being annoying, can't give a proper suggestion)

```svelte
<div class="translation-settings__keys-container">
  {#each Object.keys($orderedTranslationKeys) as key}
    <div>
       ...
    </div>
  {:else}
     <p class="m-0">No keys</p>
  {/each}
</div>
```